functions/Invoke-DbaDatabaseShrink.ps1

```diff
@@ -55,7 +55,7 @@ Specifies the method that is used to shrink the database
 Timeout in minutes. Defaults to infinity (shrinks can take a while.)
 
 .PARAMETER LogsOnly
-Only shrink the log file, not the entire database
+Deprecated. Use FileType instead
```

Comment:
Just note, we are not doing this with any other command 😸

Reply:
One that i updated the other day ;D It's required for params since you cant remove it